app/code/Magento/Inventory/view/adminhtml/ui_component/inventory_stock_form.xml

```diff
@@ -77,7 +77,7 @@
                     <item name="breakLine" xsi:type="boolean">false</item>
                 </item>
             </argument>
-            <container name="assign_sources_button" component="Magento_Ui/js/form/components/button" template="ui/form/components/button/container">
+            <container class="Magento\Inventory\Ui\Component\AssignSourcesButton" name="assign_sources_button" component="Magento_Ui/js/form/components/button" template="ui/form/components/button/container">
```

Comment:
Need to provide refactoring
More proper way is using `Button` component instead of `Container`